### PR TITLE
nimmi89/c04-iac04

### PR DESCRIPTION
**Pull request by nimmi89**
# Description

Exercise : c04-iac04

Student Name:Nimmi Varghese

> Creation of modules in terraform

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/04class/exercises/c04-iac04/nimmi89/main.tfvars [hunk 1]
```diff
@@ -0,0 +1,4 @@
+vpc_id = vpc-081afcee350babdf5
```
Comment (denstorti) [inline classes/04class/exercises/c04-iac04/nimmi89/main.tfvars:1]: This file is duplicated. It should be only inside `terraform-code` folder. Please remove it.

File: classes/04class/exercises/c04-iac04/nimmi89/variables.tf [hunk 2]
```diff
@@ -0,0 +1,25 @@
+variable "image_id" {
```
Comment (denstorti) [inline classes/04class/exercises/c04-iac04/nimmi89/variables.tf:1]: This file is duplicated, should only be inside `terraform-code` folder. Please remove it.

File: classes/04class/exercises/c04-iac04/nimmi89/main.tfvars (hunk 1 above)
Comment (nimmi89) [reply classes/04class/exercises/c04-iac04/nimmi89/main.tfvars]: Removed the files!

File: classes/04class/exercises/c04-iac04/nimmi89/variables.tf (hunk 2 above)
Comment (nimmi89) [reply classes/04class/exercises/c04-iac04/nimmi89/variables.tf]: Removed!

**Review by denstorti — APPROVED** [review]
Thanks Nimmi

**Pull request merged**